force-app/main/default/classes/SubmitCompensations_Batch_Test.cls

```diff
@@ -0,0 +1,24 @@
+@isTest
+public class SubmitCompensations_Batch_Test {
+
+    @isTest
+    public static void testIt(){
+        List<Compensation__c> comp_list = new List<Compensation__c>();
+        Compensation__c comp1 = new Compensation__c();
+        comp1.RecordTypeId = '0123h000000H5smAAC';
+        comp1.Name = 'Test Comp1';
+        comp1.Contact__c = '0033h000001shV0AAI';
```

Comment:
instead of putting this contact Id here, you could create a new contact just before creating the compensation and use its ID  =)

Reply:
thanks i will do this for the next part.